src/DependencyInjection/Factory/AdminFactoryInterface.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) 2011-2016 Symfony CMF
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Cmf\Bundle\SonataAdminIntegrationBundle\DependencyInjection\Factory;
+
+use Symfony\Component\Config\Definition\Builder\NodeBuilder;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Loader\XmlFileLoader;
+
+/**
+ * Admin factories enable new admin classes to be made available by this 
+ * integration.
+ *
+ * @author Wouter de Jong <wouter@wouterj.nl>
+ */
+interface AdminFactoryInterface
+{
+    /**
+     * The configuration key used to reference the admin.
```

Comment:
should we be explicit that this is the bundle class FQN? we rely on that in the DI extension class